app.yaml

```diff
@@ -10,3 +10,8 @@ handlers:
   static_files: build/index.html
   upload: build/index.html
   secure: always
+  http_headers:
+    X-Frame-Options: "DENY"
+    Strict-Transport-Security: "max-age=2592000; includeSubdomains"
```

Comment:
Should we look into [HSTS preloading](https://hstspreload.org/?domain=terra.bio)?

Reply:
It's a good thing to setup for, again does require some more testing since it comes with some "dangerous features" as well.